floodviz/hydrograph_utils.py

```diff
@@ -21,34 +21,32 @@ def parse_hydrodata(jdata):
     """
     all_series_data = []
     
-    if jdata is not None:
-        for idx, site in enumerate(jdata):
+    for idx, site in enumerate(jdata):
```

Comment:
Is there a reason to change this function for this task? If not, please revert so that this pull request does not make changes to this function.  If you haven't purposefully made changes you should investigate using a merge tool which will help find these issues.